01-JS-DOM-APIs/Ex04/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+		<meta charset="UTF-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+   	<title> JS 04 </title>
+   	<meta name="description" content="Die promises, die">
+   	<meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Bubbler+One" rel="stylesheet">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <header>
+     	<h1>This it my fourth JS excercise!</h1>
+    </header>
+    <div class="repWrapper">
+      <h2>Repositories:</h2>
+      <ul id="repoList">
+      </ul>
+    </div>
+    <div class="searchBox">
+      <form name="search">
+        <label>Search a repository:</label>
+        <input type="text" id="repoSearch">
+        <input type="submit" value="Search" onclick="qSearch(document.getElementById('repoSearch').value)">
```

Comment:
You can use the parameter in javascript directly, there is no need to provide it here. Did you tried that?